pkg/grafana/rules.go

```diff
@@ -52,7 +52,7 @@ func getRemoteRuleGroup(uid string) (*grizzly.Resource, error) {
 						"rules": group.Rules,
 					}
 					handler := RuleHandler{}
-					resource := grizzly.NewResource(handler.APIVersion(), handler.Kind(), uid, spec)
+					resource := grizzly.NewResource(handler.APIVersion(), handler.Kind(), group.Name, spec)
```

Comment:
Reeeeeeeeally nit, but we can use `name` instead of `group.Name` since we have already extracted the var